model/Core/Classes/IndividualElement.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# IndividualElement
+
+## Summary
+
+A concrete subclass of Element used by Individuals in the
+Core profile.
+
+## Description
+
+Individuals, such as NoneElement and NoAssertionElement, need to reference a
+concrete subclass of ELement.
+
+This class provides the type used by the individuals.
```

Comment:
```suggestion
This class provides the type used by the individuals.

For individuals that are subclass of AnyLicenseInfo, please use IndividualLicensingInfo.
```

As AnyLicenseInfo is also a subclass of Element, do we have to point that out here as well?

Reply:
We shouldn't be talking about individuals defined in other profiles here.